server/renderer.js

```diff
@@ -1,363 +1,172 @@
 "use strict";
 
-var fs              = require("fs")
+var _fs             = require("fs")
 var path            = require("path")
 
 var config          = require("../config")
 var logger          = require("./logger")
 var deasync         = require("deasync")
 var extend          = require("extend")
-
-var TEMPLATES_DIR   = path.join(config.paths.client, "/hbs/")
+let denodeify       = require("denodeify")
```

Comment:
I mean, technically all of them could be `const`... I'm not really sure what convention to stick with here.  Ideally we'd use ES6 `imports` and not have to care but Node doesn't do those yet.